keras_contrib/applications/densenet.py

```diff
@@ -399,10 +403,11 @@ def DenseNetFCN(input_shape, nb_dense_block=5, growth_rate=16, nb_layers_per_blo
         else:
             img_input = input_tensor
 
-    x = __create_fcn_dense_net(classes, img_input, include_top, nb_dense_block,
-                               growth_rate, reduction, dropout_rate, weight_decay,
+    x = __create_fcn_dense_net(classes, img_input, include_top, nb_dense_block, growth_rate,
+                               reduction, dropout_rate, weight_decay,
                                nb_layers_per_block, upsampling_conv, upsampling_type,
-                               init_conv_filters, input_shape, activation)
+                               init_conv_filters, input_shape, activation,
+                               early_transition, transition_pooling, initial_kernel_size)
```

Comment:
`transition_pooling` seems to be undefined uptil here. Perhaps you meant to rename the `pooling='max'` to this name?

Reply:
fixed